src/main/java/duke/memo/message/MessageGenerator.java

```diff
@@ -0,0 +1,78 @@
+package duke.memo.message;
+
+import duke.memo.data.TaskList;
+import duke.memo.task.Task;
+
+import java.util.ArrayList;
+
+public class MessageGenerator {
+
+    public MessageGenerator() {
```

Comment:
Could consider having the methods as static, since the methods do not depend on any instance variables